test/over_react_redux/value_mutation_checker_test.dart

```diff
@@ -22,7 +22,7 @@ import 'package:test/test.dart';
 main() {
   group('Value Mutation Checker:', () {
     group('CollectionLengthHasher', () {
-      sharedHashTests(useLengthHasher: false);
```

Comment:
This should have been `true`, so shared tests for this class weren't actually being run